08-transport.md

```diff
@@ -609,7 +609,243 @@ It is strongly recommended that existing, commonly-used, validated
+    ls.priv: 0x1111111111111111111111111111111111111111111111111111111111111111
+    ls.pub: 0x034f355bdcb7cc0af728ef3cceb9615d90684bb5b2ca5f859ab0f0b704075871aa
+    e.priv: 0x1212121212121212121212121212121212121212121212121212121212121212
+    e.pub: 0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7
+    output: 0x00036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f70df6086551151f58b8afe6c195782c6a
+    # Act Two
+    input: 0x0102466d7fcae563e5cb09a0d1870bb580344804617879a14949cf22285f1bae3f276e2470b93aac583c9ef6eafca3f730ae
+    output: ERROR (ACT2_BAD_VERSION 1)
+
+    name: transport-initiator act2 bad key serialization test
+    rs.pub: 0x028d7500dd4c12685d1f568b4c2b5048e8534b873319f3a8daa612b469132ec7f7
+    ls.priv: 0x1111111111111111111111111111111111111111111111111111111111111111
+    ls.pub: 0x034f355bdcb7cc0af728ef3cceb9615d90684bb5b2ca5f859ab0f0b704075871aa
+    e.priv: 0x1212121212121212121212121212121212121212121212121212121212121212
+    e.pub: 0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7
+    output: 0x00036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f70df6086551151f58b8afe6c195782c6a
+    # Act Two
+    input: 0x0004466d7fcae563e5cb09a0d1870bb580344804617879a14949cf22285f1bae3f276e2470b93aac583c9ef6eafca3f730ae
+    output: ERROR (ACT2_BAD_PUBKEY)
+
+    name: transport-initiator act2 bad MAC test
+    rs.pub: 0x028d7500dd4c12685d1f568b4c2b5048e8534b873319f3a8daa612b469132ec7f7
+    ls.priv: 0x1111111111111111111111111111111111111111111111111111111111111111
+    ls.pub: 0x034f355bdcb7cc0af728ef3cceb9615d90684bb5b2ca5f859ab0f0b704075871aa
+    e.priv: 0x1212121212121212121212121212121212121212121212121212121212121212
+    e.pub: 0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7
+    output: 0x00036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f70df6086551151f58b8afe6c195782c6a
+    # Act Two
+    input: 0x0002466d7fcae563e5cb09a0d1870bb580344804617879a14949cf22285f1bae3f276e2470b93aac583c9ef6eafca3f730af
+    output: ERROR (ACT2_BAD_TAG)
+
+## Responder Tests
+
+The responder should produce the given output when fed this input.
+
+    name: transport-responder successful handshake
+    ls.priv=2121212121212121212121212121212121212121212121212121212121212121
+    ls.pub=028d7500dd4c12685d1f568b4c2b5048e8534b873319f3a8daa612b469132ec7f7
+    e.priv=0x2222222222222222222222222222222222222222222222222222222222222222
+    e.pub=0x02466d7fcae563e5cb09a0d1870bb580344804617879a14949cf22285f1bae3f27
+    # Act One
+    input: 0x00036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f70df6086551151f58b8afe6c195782c6a
+    # re=0x036360e856310ce5d294e8be33fc807077dc56ac80d95d9cd4ddbd21325eff73f7
+    # h=0x9e0e7de8bb75554f21db034633de04be41a2b8a18da7a319a03c803bf02b396c
+    # ss=0x1e2fb3c8fe8fb9f262f649f64d26ecf0f2c0a805a767cf02dc2d77a6ef1fdcc3
```

Comment:
I believe that you've used secp256k1's ecdh function which returns sha256(serialized compressed point) instead of point.x  ? With this change I can generate yours result with our own implementation.